be/src/exec/spill/spiller.h

```diff
@@ -46,19 +46,46 @@ struct SpillProcessMetrics {
     SpillProcessMetrics() = default;
     SpillProcessMetrics(RuntimeProfile* profile);
 
+    std::shared_ptr<RuntimeProfile> _spiller_metrics;
+
+    // time spent to append data into Spiller
     RuntimeProfile::Counter* spill_timer = nullptr;
+    // the number of rows appended to Spiller
     RuntimeProfile::Counter* spill_rows = nullptr;
+    // time spent to flush data to disk
     RuntimeProfile::Counter* flush_timer = nullptr;
-    RuntimeProfile::Counter* restore_timer = nullptr;
+    // disk io time during flush
     RuntimeProfile::Counter* write_io_timer = nullptr;
+    // time spent to restore data from Spiller, which includes the time to try to get data from buffer and drive the next prefetch
+    RuntimeProfile::Counter* restore_timer = nullptr;
+    // disk io time during restore
+    RuntimeProfile::Counter* read_io_timer = nullptr;
+    // the number of rows restored from Spiller
     RuntimeProfile::Counter* restore_rows = nullptr;
+    // time spent to shuffle data to the corresponding partition, only used in join operator
     RuntimeProfile::Counter* shuffle_timer = nullptr;
+    // time spent to split partitions, only used in join operator
     RuntimeProfile::Counter* split_partition_timer = nullptr;
 
+    // date bytes flushed to disk
     RuntimeProfile::Counter* flush_bytes = nullptr;
+    // data bytes restored from disk
     RuntimeProfile::Counter* restore_bytes = nullptr;
+    // time spent to serialize data before flush it to disk
     RuntimeProfile::Counter* serialize_timer = nullptr;
+    // time spent to deserialize data after read it from disk
     RuntimeProfile::Counter* deserialize_timer = nullptr;
+
+    // data bytes restored from mem table in memory, only used in join operator
+    RuntimeProfile::Counter* restore_from_mem_bytes = nullptr;
```

Comment:
how does it differ from `resotre_bytes`?

Reply:
> how does it differ from `resotre_bytes`?

restore_bytes is all data bytes restored from disk, for hash join operators, some partition may be kept in memory , these data will be read from mem table directly during restore and record in restore_from_mem_bytes.

Reply:
`estore_from_mem_bytes`the _mem_  is OK? may be confuse with memory, prefer estore_from_mem_table_bytes or estore_from_buffer_bytes

Reply:
> `estore_from_mem_bytes`the _mem_ is OK? may be confuse with memory, prefer estore_from_mem_table_bytes or estore_from_buffer_bytes

fixed. renamed them to restore_from_mem_table_xx